index.html

```diff
@@ -0,0 +1,191 @@
+        Functions can be properties of objects, executing as loosely typed methods.</p>
+    <ul>
+        <li>They are similar in some ways</li>
+        <li>They are fundamentally different in some ways</li>
+    </ul>
+  </section>
+  <section class="main-section" id="Hello_world"> 
+    <header>
+      Hello world
+    </header>
+    <code>function greetMe(yourName) { alert("Hello " + yourName); }
+        greetMe("World");</code>
+  </section>
+  <section class="main-section" id="Variables">
+    <header>
+      Variables
+    </header>
+    <p>You use variables as symbolic names for values in your application. The names of variables, 
+        called identifiers, conform to certain rules.</p>
+    </section>
+  <section class="main-section" id="Declaring_variables"> 
+    <header>
+      Declaring variables
+    </header>
+    <p>You can declare a variable in three ways:
+    </p>
+    <code>var x = 42.</code>
+    <code>x = 42.</code>
+    <code>let y= 13.</code>
+  </section>
+   <section class="main-section" id="Variables_scope"> 
+    <header>
+      Variables scope
+    </header>
+    <p>When you declare a variable outside of any function, it is called a global variable, 
+        because it is available to any other code in the current document. When you declare 
+        a variable within a function, it is called a local variable, because it is available only 
+        within that function.
+    </p>
+    <code>if (true) { var x = 5; } console.log(x); // 5 --Before 2015</code>
+    <code>if (true) { let y = 5; } console.log(y); // ReferenceError: y is
+        not defined -- ECMAScript 2015</code>
+  </section>
+   <section class="main-section" id="Global_variables"> 
+    <header>
+      Global variables
+    </header>
+    <p>Global variables are in fact properties of the global object. In web pages the 
+        global object is window, so you can set and access global variables using 
+        the window.variable syntax.</p>
+
+    <p>Consequently, you can access global variables declared in one window or frame from another window or frame by specifying the window or frame name. For example, if a variable called phoneNumber is declared in a document, 
+        you can refer to this variable from an iframe as parent.phoneNumber.</p>
+  
+  </section>
+   <section class="main-section" id="Constants"> 
+    <header>
+      Constants
+    </header>
+    <p>You can create a read-only, named constant with the const keyword. 
+        The syntax of a constant identifier is the same as for a variable identifier: 
+        it must start with a letter, 
+        underscore or dollar sign and can contain alphabetic, numeric, or underscore characters.</p>
+    <code>const PI = 3.14;</code>
+  </section>
+   <section class="main-section" id="Data_types"> 
+    <header>
+      Data types
+    </header>
```

Comment:
✔️ Good job done using Ids. But it seems your code has so many ids. 🤔 I think just one or two is okay for a page. Many ids can create problems for managing styles in one place
id and header issues

✔️ Good semantics job by using header. But some headers seems to have no style class. 🤔 I think If you have others pages where headers change it will be difficult to manage. Kindly work on it.
id and header issues2